src/VueDatePicker/composables/position.ts

```diff
@@ -225,7 +225,7 @@ export const usePosition = ({
     const getScrollableParent = function (el: HTMLElement | null): Window | HTMLElement {
         if (!el || el === document.body || el.nodeType === Node.DOCUMENT_FRAGMENT_NODE) return window;
         if (isScrollable(el)) return el;
-        return getScrollableParent(el.parentNode as HTMLElement);
+        return getScrollableParent(el.assignedSlot ? el.assignedSlot.parentElement : el.parentElement);
```

Comment:
@biesbjerg Can you replace `: el.parentElement` with` : el.parentNode as HTMLElement` on the else part? As it breaks close menu transition.

Reply:
Can do! I've changed it to `parentNode` for both code paths for consistency's sake (`getScrollableParent((el.assignedSlot ? el.assignedSlot.parentNode : el.parentNode) as HTMLElement)`)

For what it's worth, close menu transition seemed to work fine for me with `parentElement` as well.

![Untitled5](https://github.com/Vuepic/vue-datepicker/assets/230182/099980ed-6664-4a34-94a3-c7dd235e370f)